core/client/app/adapters/base.js

```diff
@@ -28,7 +28,7 @@ export default RESTAdapter.extend({
 
     buildURL(type, id) {
```

Comment:
The `type, id` arguments need to be removed here so that our linting passes, it's complaining because we define the vars but never use them because we're now using the spread operator `...arguments`.